File: specification/specs/x_pack/machine_learning/post_calendar_events/ScheduledEvent.ts
class ScheduledEvent {
	calendar_id: Id;
	description: string;
	@prop_serializer("NullableDateTimeOffsetEpochMillisecondsFormatter")
	start_time: Date;
	@prop_serializer("NullableDateTimeOffsetEpochMillisecondsFormatter")
	end_time: Date;
	event_id: Id;
}

File: output/csharp/XPack/MachineLearning/PostCalendarEvents/ScheduledEvent.cs
using Nest.CommonAbstractions;
using Nest.Internal;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class ScheduledEvent  {
		
		[DataMember(Name="calendar_id")]
		public Id CalendarId { get; set; }


		[DataMember(Name="description")]
		public string Description { get; set; }


		[DataMember(Name="end_time")]
		public DateTimeOffset EndTime { get; set; }


		[DataMember(Name="event_id")]
		public Id EventId { get; set; }


		[DataMember(Name="start_time")]
		public DateTimeOffset StartTime { get; set; }

	}
}
